draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -360,18 +414,20 @@ This is to align the Router Lifetime with the recommendations in <xref target="R
 </t>
 </list>
 </t>
-
-
-
-
-
+<t>
+While allowing updates to the valid lifetime in RAs could enable an attacker to invalidate addresses by setting the valid lifetime to zero, this doesn't significantly worsen the security situation. An attacker capable of sending rogue RAs already has the power to disrupt connectivity by manipulating other parameters, such as gateway or DNS information.  Therefore, accepting a zero lifetime doesn't make the system more vulnerable than it already is in the presence of such an attacker.
+</t>
+<t>
+In scenarios where RA-based attacks are of concern, proper mitigations such as RA-Guard <xref target="RFC6105"/> <xref target="RFC7113"/> or SEND <xref target="RFC3971"/> should be implemented. 
+</t>
+<t>=======</t>
 
 <t>
 <list style="hanging">
 <t hangText="RATIONALE:">
 <list style="symbols">
 <t>
-This change allows hosts to react to the signal provided by a router that has positive knowledge that a prefix has become invalid.
+This change allows hosts to react to the signal provided by a router that has positive knowledge that a prefix is not assigned to the given link anymore. In particular it would allow the host to invalidate addresses from that prefix, and, if the prefix is reassigned to another link, allows the host to communicate to devices on that link.
```

Comment:
FWIW, just to note that I like this one! -- the text is better than the original ;=)

Reply:
;) Glad to hear ;)